dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -180,7 +180,6 @@ def unstart
   # POST /api/v1/pd/workshops/1/end
   def end
     @workshop.end!
-    Pd::AsyncWorkshopHandler.process_ended_workshop @workshop.id
```

Comment:
Wait... we were doing this synchronously _anyway_? :joy:

Reply:
I don't think so, from my reading of the code- in production/test this would have invoked [`SQS::SQSQueue#enqueue`](https://github.com/code-dot-org/code-dot-org/pull/29903/files#diff-4c5ff9f864d342923ae1901514021926L23).

Reply:
Oh, my bad, I didn't read carefully.  I assumed `process_ended_workshop` meant what it said!